client/src/components/Recipe.tsx

```diff
@@ -1,10 +1,14 @@
 // Package imports
+import React from 'react';
 import { Card, Badge } from 'react-bootstrap';
 
 export default function Recipe ({ recipe }) {
+
+  // console.log(recipe);
+
```

Comment:
Can we remove this? Don't want some of our feedback to come back including "No commented out lines of code" when Alistair/Vic can spend their time giving us actual feedback.

Reply:
done